### ISSUE TITLE
PHP 7.4 Deprecated: Array and string offset access syntax with curly braces is deprecated

### ISSUE THREAD

**Issue by Mickroz** [planetteamspeak/ts3phpframework#141]
getting this with PHP 7.4
```
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/String.php on line 924
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/String.php on line 934
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Node/Host.php on line 542
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Node/Host.php on line 542
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Crypt.php on line 177
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Char.php on line 182
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Char.php on line 194
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Char.php on line 198
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Char.php on line 198
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Char.php on line 202
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Char.php on line 202
Deprecated: Array and string offset access syntax with curly braces is deprecated in /libraries/TeamSpeak3/Helper/Char.php on line 202
```

**Comment by Fjonan** [comment]
Created a pull request to fix that: #142

**Comment by ronindesign** [comment]
Merged. Thank you for your time on this update and the pull request, appreciate it!
